opencadd/structure/superposition/api.py

```diff
@@ -15,7 +15,7 @@
 }
 
 
-def align(structures, method=TheseusAligner, **kwargs):
+def align(structures, method=TheseusAligner, only_backbone=False, **kwargs):
```

Comment:
The method option "only_backbone" will be removed in PR #115. 
It can be done with the selection string in PR #115 .
If you want, I can add it to the docstring though if it is still desired.

Reply:
Perfect - I prefer the selection string.